src/ColorGrid.js

```diff
@@ -14,56 +14,60 @@ const ColorGrid = ({ hueValue, saturationCount, luminosityCount }) => {
 
   return (
     <div
-      className='colorgrid'
-      style={{ border: '1px', borderBlock: '10px', borderColor: 'red' }}
+      className="colorgrid"
+      style={{ border: "1px", borderBlock: "10px", borderColor: "red" }}
     >
       {
-        // create saturationCount divs
-        Array(saturationCount - 1)
-          .fill(0)
-          .map((_, i) => {
-            return (
-              <div key={i} style={{ display: 'flex', flexDirection: 'row' }}>
-                {
-                  // create luminosityCount divs
-                  Array(luminosityCount - 1)
-                    .fill(0)
-                    .map((_, j) => {
-                      var hueLocal = hueValue;
-                      var saturationLocal = j * saturationFraction;
-                      var luminosityLocal = j * luminosityFraction;
-                      const hexColour = hsl.hex(hueLocal,saturationLocal,luminosityLocal);
-                      const styles = reactCSS({
-                        default: {
-                          color: {
-                            width: `${oneColorWidth}px`,
-                            height: `${oneColorHeight}px`,
-                            borderRadius: '5px',
-                            background: `hsl(${hueLocal}, ${saturationLocal}%, ${luminosityLocal}%)`,
-                            color: luminosityLocal > 50 ? "#000" : "#fff",
-                          },
-                          swatch: {
-                            padding: '1px',
-                            background: '#fff',
-                            borderRadius: '5px',
-                            boxShadow: '0 0 0 1px rgba(0,0,0,.1)',
-                            display: 'inline-block',
-                            cursor: 'pointer',
-                          },
-                        },
-                      });
-                      return (
-                        <div key={j} style={styles.swatch}>
-                          <div style={styles.color} className="colour-cell" onClick={() => {navigator.clipboard.writeText(hexColour)}}>
-                            <p>{hexColour}</p>
-                          </div>
-                        </div>
-                      );
-                    })
-                }
-              </div>
-            );
-          })
+        // create saturationCount div
+        <div style={{ display: "flex", flexDirection: "row" }}>
+          {
+            // create luminosityCount divs
+            Array(luminosityCount - 1)
+              .fill(0)
+              .map((_, i) => {
+                var hueLocal = hueValue;
+                var saturationLocal = i * saturationFraction;
+                var luminosityLocal = i * luminosityFraction;
```

Comment:
You changed both, one needs to be i and other needs to be j

Reply:
And needs to be grid, not a single row

Reply:
Got it 👍🏼 
Will fix that.